core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/store/dot-experiments-list-store.ts

```diff
@@ -147,6 +181,45 @@ export class DotExperimentsListStore
         );
     });
 
+    readonly addExperiments = this.effect(
+        (experiment$: Observable<Pick<DotExperiment, 'pageId' | 'name' | 'description'>>) => {
+            return experiment$.pipe(
+                tap(() => this.setSidebarStatus({ status: ComponentStatus.SAVING, isOpen: true })),
+                switchMap((experiment) =>
+                    this.dotExperimentsService.add(experiment).pipe(
+                        tapResponse(
+                            (experiment) => {
+                                this.messageService.add({
+                                    severity: 'info',
+                                    summary: this.dotMessageService.get(
+                                        'experiments.action.add.confirm-title'
+                                    ),
+                                    detail: this.dotMessageService.get(
+                                        'experiments.action.add.confirm-message',
+                                        experiment.name
+                                    )
+                                });
+                                this.addExperiment(experiment);
+
+                                this.router.navigate(
+                                    [
+                                        '/edit-page/experiments/configuration',
+                                        experiment.pageId,
+                                        experiment.id
+                                    ],
+                                    {
+                                        queryParamsHandling: 'preserve'
+                                    }
+                                );
+                            },
+                            (error: HttpErrorResponse) => throwError(error)
```

Comment:
handle error with `this.dotHttpErrorManagerService.handle(error)` and add test

Reply:
Added